src/pybind/mgr/dashboard/tests/test_host.py

```diff
@@ -123,14 +123,14 @@ def test_host_list_with_facts(self, mock_get_hosts):
             self._get('{}?facts=true'.format(self.URL_HOST), version=APIVersion(1, 1))
             self.assertStatus(200)
             self.assertHeader('Content-Type',
-                              'application/vnd.ceph.api.v1.1+json')
+                              'application/vnd.ceph.api.v1.2+json')
```

Comment:
When replacing code, let's use the improved code/helpers available:
```suggestion
                              APIVersion(1,2).to_mime_type())
```

Reply:
done this as well.